lib/cloudkeeper/aws/cloud.rb

```diff
@@ -111,9 +121,8 @@ def timeout
       # @note This method can be billed by AWS
       # @param image_id [String] id of specific AMI
       def deregister_image(image_id)
-        ec2.deregister_image(
-          image_id: image_id
-        )
+        logger.debug { "Deregistering image ##{image_id}" }
```

Comment:
What is `image_id` in this context? An AMI's ID? If so, please use it in the log to distinguish between bucket image and AMI image. Same goes for all AMI specific operations and their logging.